buildSrc/src/main/java/Dependencies.kt

```diff
@@ -148,8 +154,11 @@ object Dependencies {
     }
 
     object Protobuf {
+        @Deprecated("A new implementation is avaliable. Use the protobuf project instead.")
```

Comment:
Why aren't we deleting this? The implementation has been deleted, no?

Reply:
Still used in Network for envelope serialisation.

I want to do that in another PR.